legal-api/src/legal_api/core/filing.py

```diff
@@ -460,7 +460,14 @@ def _add_ledger_order(filing: FilingStorage, ledger_filing: dict) -> dict:
     @staticmethod
     def _is_display_ledger(filing: FilingStorage) -> bool:
         """Return boolean that display the ledger."""
-        return filing.filing_type != Filing.FilingTypes.ADMIN_FREEZE
+        # If filing is NOT an admin freeze or involuntary dissolution, we will display it on ledger
+        return not (
+            filing.filing_type == Filing.FilingTypes.ADMIN_FREEZE or
+            (
+                filing.filing_type == Filing.FilingTypes.DISSOLUTION and
+                filing.filing_sub_type == 'involuntary'
```

Comment:
I was hoping to use an ENUM for the involuntary but saw that the string was being used everywhere as well. I was thinking of changing it everywhere to use an enum.

Reply:
I think it makes sense to use an enum, if it isn't too much effort to make the same changes everywhere else

Reply:
I'm just going to leave it for now. If this pops up again, might be a good idea to create an enum then.